src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const result = {};
+  const strings = sourceString.split(';').map((str) => str.trim());
```

Comment:
The chaining of methods here is correct, but be mindful of the indentation when chaining multiple methods for better readability. In this case, the code is simple enough that it's acceptable.